docs/irods.css

```diff
@@ -94,3 +94,8 @@ a:hover {
 .rst-content .section ol p, .rst-content .section ul p {
     margin-bottom: 3px;
 }
+span.hljs-comment {
+    color: #6f99c3;
+    font-style: italic;
```

Comment:
i think this looks better with `normal`, rather than `italic`.

?

Reply:
This styling came from the tool that generated the CSS. I only changed the color.

Will change and post what it looks like. You can open the developer tools and play with the colors and things in realtime. Just an fyi.

Reply:
Normal looks fine to me. Will use that.

Reply:
done